sql/postgres/postgres.go

```diff
@@ -10,17 +10,28 @@ import (
 	_ "github.com/lib/pq" //postgres driver
 )
 
+var (
+	// AcceptTableRecreationWarning indicates that you accept all consequences from dropping your table and recreating it
+	AcceptTableRecreationWarning = "destructive warning accepted"
+)
+
 // Options are the postgres datastore options, reexported here for convenience.
 type Options struct {
-	Host           string
-	Port           string
-	User           string
-	Password       string
-	Database       string
-	Table          string
-	SSLMode        string
-	RunMigrations  bool
-	RecreateTables bool
+	// must be set to "warning is accepted"
+	// this will prevent anyone from accidentally nuking their database
+	// the act of a specific string ensures that its more difficult
+	// to take this action
+	AcceptRecreateWarning string
```

Comment:
Comment for `AcceptRecreateWarning` does not match value for `AcceptTableRecreationWarning`

Also, one name has the word table in it while the other does not, I think this is confusing.

Reply:
good idea, i cleaned up the naming abit